attr.go

```diff
@@ -27,7 +27,7 @@ const (
 	AttrUnderline
 	AttrDim
 	AttrItalic
-	AttrStrikeOut
+	AttrStrikethrough
```

Comment:
Please rename this to AttrStrikeThrough   (strike-through is two words or hyphenated)